src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -196,12 +196,32 @@ If you started your project using [the `create astro` wizard](/en/install/auto/#
 
 Runs Astro's development server. This is a local HTTP server that doesn't bundle assets. It uses Hot Module Replacement (HMR) to update your browser as you save changes in your editor.
 
+<h3>Flags</h3>
+
+Use these flags to customize the behavior of the command.
+
+#### `--viteMode <mode>`
+
+It defines [mode](https://vitejs.dev/config/shared-options.html#mode) property of the internal Vite config.
+
+<ReadMore>Read more about [Vite modes](https://vitejs.dev/guide/env-and-mode#modes).</ReadMore>
+
 ## `astro build`
 
 Builds your site for deployment. By default, this will generate static files and place them in a `dist/` directory. If [SSR is enabled](/en/guides/server-side-rendering/), this will generate the necessary server files to serve your site.
 
 Can be combined with the [common flags](#common-flags) documented below.
 
+<h3>Flags</h3>
+
+Use these flags to customize the behavior of the command.
+
+#### `--viteMode <mode>`
+
+It defines [mode](https://vitejs.dev/config/shared-options.html#mode) property of the internal Vite config.
+
+<ReadMore>Read more about [Vite modes](https://vitejs.dev/guide/env-and-mode#modes).</ReadMore>
```

Comment:
```suggestion
#### `--viteMode <mode>`
<p><Since v="4.8.0" /></p>

Configures the [`mode`](https://vitejs.dev/config/shared-options.html#mode) property of the internal Vite config. Defaults to `production`.

<ReadMore>Read more about [Vite modes](https://vitejs.dev/guide/env-and-mode#modes).</ReadMore>
```
Whichever word you choose above, use the same one here!